Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1516,6 +1496,245 @@ def broadcast_message():
         return_error('No messages were sent. Verify that the inputs are correct.')
 
 
+def shutdown_agents_request(query, agent_id, group_id):
+    endpoint_url = 'agents/actions/shutdown'
+    filters = {}
+
+    if query:
+        filters['query'] = query
+    if agent_id:
+        filters['ids'] = agent_id
+    if group_id:
+        filters['groupIds'] = group_id
+
+    payload = {
+        'filter': filters
+    }
+
+    response = http_request('POST', endpoint_url, data=json.dumps(payload))
+    if response.get('errors'):
+        return_error(response.get('errors'))
+    else:
+        return response
+
+
+def shutdown_agents():
+    """
+    Sends a shutdown command to all agents matching the input filter
+    """
+    query = demisto.args().get('query', '')
+
+    agent_id = argToList(demisto.args().get('agent_id'))
+    group_id = argToList(demisto.args().get('group_id'))
+    if not (agent_id or group_id):
+        return_error('Expecting at least one of the following arguments to filter by: agent_id, group_id.')
+
+    affected_agents = shutdown_agents_request(query, agent_id, group_id)
+    agents = affected_agents.get('data', {}).get('affected', 0)
+    if agents > 0:
+        demisto.results(f'Shutting down {agents} agent(s).')
+    else:
+        return_error('No agents were shutdown.')
+
+
+def uninstall_agent_request(query=None, agent_id=None, group_id=None):
+    endpoint_url = 'agents/actions/uninstall'
+    filters = {}
+
+    if query:
+        filters['query'] = query
+    if agent_id:
+        filters['ids'] = agent_id
+    if group_id:
+        filters['groupIds'] = group_id
+
+    payload = {
+        'filter': filters
+    }
+
+    response = http_request('POST', endpoint_url, data=json.dumps(payload))
+    if response.get('errors'):
+        return_error(response.get('errors'))
+    else:
+        return response
+
+
+def uninstall_agent():
+    """
+    Sends an uninstall command to all agents matching the input filter.
+    """
+    query = demisto.args().get('query', '')
+
+    agent_id = argToList(demisto.args().get('agent_id'))
+    group_id = argToList(demisto.args().get('group_id'))
+    if not (agent_id or group_id):
+        return_error('Expecting at least one of the following arguments to filter by: agent_id, group_id.')
+
+    affected_agents = shutdown_agents_request(query, agent_id, group_id)
+    agents = affected_agents.get('data', {}).get('affected', 0)
+    if agents > 0:
+        demisto.results(f' Uninstall was sent to {agents} agent(s).')
```

Comment:
please advice with @kirbles19 on that message

Reply:
Took this line from the UI, But I'll ask him